plugins/woocommerce/src/StoreApi/Routes/V1/Checkout.php

```diff
@@ -198,7 +228,15 @@ public function validate_required_additional_fields( \WP_REST_Request $request )
 				}
 			}
 		}
+	}
 
+	/**
+	 * Validate required address fields.
+	 *
+	 * @param \WP_REST_Request $request Request object.
+	 * @throws RouteException When a required address field is missing.
+	 */
+	private function validate_required_additional_fields_for_customer( \WP_REST_Request $request ) {
```

Comment:
I split the `validate_required_additional_fields` function into two here, to be able to validate customer and order fields separately.